app/user.go

```diff
@@ -1858,22 +1856,20 @@ func (a *App) AutocompleteUsersInChannel(teamId string, channelId string, term s
 
 	if !a.IsESAutocompletionEnabled() || err != nil {
 		autocomplete = &model.UserAutocompleteInChannel{}
-		uchan := a.Srv.Store.User().SearchInChannel(channelId, term, options)
```

Comment:
Here the approach would be to use the "explicit async" approach, creating a channel  and running it in a gorouting. The idea is to run this function and the `SearchNotInChannel` in a concurrent way. you can see an example of the "explicit async" approach in the example PR referenced in the issue.

Reply:
Ah, sorry, didn't realise this approach was required. I'll update the PR. Thanks!